source/_integrations/knx.markdown

```diff
@@ -75,9 +75,80 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+### KNX Connection
+
+Select the connection type to your KNX bus. The integration supports the following connection methods:
+
+- 'Automatic' performs a gateway scan on start, to find a KNX IP interface. It will connect via a tunnel. This option is not available when a gateway scan at setup initiation was not successful.
+- 'Tunneling' will connect to a specific KNX IP interface over a tunnel.
+- 'Routing' will use Multicast to communicate with KNX IP routers.
+
+For more information about KNX connection types see [Connection](#connection).
+
+### Tunneling specific settings
+
+You can select a specific tunnel device if multiple were found during the setup. In case no device was discovered you can configure connection information manually.
```

Comment:
```suggestion
You can select a specific tunnel device if multiple were found during the setup. In case no device was discovered, you can configure connection information manually.
```